x-pack/performance/tsconfig.json

```diff
@@ -12,5 +12,7 @@
     "@kbn/journeys",
     "@kbn/test-subj-selector",
     "@kbn/tooling-log",
+    "@kbn/test",
+    "@kbn/expect",
```

Comment:
Do we need to have `@kbn/test` ?

Reply:
yes, for [this part](https://github.com/elastic/kibana/pull/151613/files#diff-058de98af983638f8ae17761d683a257450ad07727dd147a358fabd987129c43R8).